### DISCUSSION TITLE
Form View -> Description formatting

### DISCUSSION THREAD

**Discussion by divemasterza** [nocodb/nocodb#7038] [Q&A] [answered]
Is there any way to format the the description field in the form view?
(bolds, line returns, italics etc;)

**Comment by dstala** [comment, marked as answer]
Not supported currently. We are working on to bring in Rich text formatting to Long-text field, but not thought about form view yet.

**Comment by divemasterza** [comment]
Would be a useful feature :) especially for public facing form/survery, where a description is needed :)
